build.sh

```diff
@@ -52,7 +52,7 @@ if [[ ! -f ${STARROCKS_THIRDPARTY}/installed/include/fast_float/fast_float.h ]];
     ${STARROCKS_THIRDPARTY}/build-thirdparty.sh
 fi
 
-WITH_BLOCK_CACHE=OFF
+WITH_BLOCK_CACHE=ON
```

Comment:
I think we'd better keep the switch to off default.

Reply:
yes. sorry, I don't intend to enable it.